### DISCUSSION TITLE
about duplicates

### DISCUSSION THREAD

**Discussion by scarf005** [realworld-apps/realworld#1227] [Q&A] [answered]
i was thinking of doing `typescript` + `react` + `react query` + `react router` + `jotai`, 
however #1148 already does `typescript` + `react` + `react query` + `react-dom-router`.

would it be considered duplicate? or would there be any guidelines on how smiliar is duplicates?

**Comment by geromegrignon** [comment, marked as answer]
Hello, given the popularity of jotai, doing a proper implementation is fine with me.
